src/DependencyInjection/Configuration.php

```diff
@@ -15,13 +15,7 @@ class Configuration implements ConfigurationInterface
     public function getConfigTreeBuilder()
     {
         $treeBuilder = new TreeBuilder('qp_woohoolabs_yin');
-
-        if (\method_exists($treeBuilder, 'getRootNode')) {
-            $rootNode = $treeBuilder->getRootNode();
-        } else {
-            // BC layer for symfony/config 4.1 and older
-            $rootNode = $treeBuilder->root('qp_woohoolabs_yin');
-        }
+        $rootNode = $treeBuilder->getRootNode();
```

Comment:
Is it already time to break compatibility with `sensio/framework-extra-bundle@5.4.1` and below? Or do you maybe want to keep this around a bit longer?
I don't know if everyone is already ready to upgrade...